main.py

```diff
@@ -171,10 +173,33 @@ def get_scores(league, time):
     print_league_scores(fixtures_results)
 
 
+def supported_leagues(total_data):
+    """ Filters out scores of unsupported leagues """
+
+    supported_leagues = {val: key for key, val in LEAGUE_IDS.items()}
```

Comment:
@ueg1990 I left this in here so that I can get the name of the league. I'll remove it as soon as we can come up with a better solution.